web_console/dynamic-plug-ins.adoc

```diff
@@ -0,0 +1,30 @@
+:_content-type: ASSEMBLY
+[id=" adding-dynamic-plugin-to-console"]
+= Adding a dynamic plug-in to the {product-title} web console
+include::modules/common-attributes.adoc[]
+:context: dynamic-plugins
+
+toc::[]
+
+You can create and deploy a dynamic plug-in on your cluster that is loaded at run-time.
+
+:FeatureName: Dynamic plug-ins
+include::snippets/technology-preview.adoc[leveloffset=+1]
+//
+
+include::modules/about-dynamic-plug-ins.adoc[leveloffset=+1]
+
+include::modules/dynamic-plug-ins-get-started.adoc[leveloffset=+1]
+
+include::modules/adding-new-extension-dynamic-plug-in.adoc[leveloffset=+1]
+
+include::modules/running-your-dynamic-plug-in.adoc[leveloffset=+1]
```

Comment:
You probably want to run your plugin to see it working before adding a new extension.